src/elements/message/message.js

```diff
@@ -1,15 +1,20 @@
 import './message.less'
-// import React, { useState, useEffect, useRef } from 'react'
 import React from 'react'
-// import url from '/util/url'
+// import Linkify from 'react-linkify'
```

Comment:
Don't think we need to keep remnants of a library that didn't work around.

Reply:
✔️